index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>My Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <p id="time">9:41</p>
+      <div class="header1">
```

Comment:
[OPTIONAL]

- KIndly remove the mobile phone icons as its not part of the design. Its just to show that the design is for mobile and not desktop

![image](https://user-images.githubusercontent.com/63473584/212179726-26a6b829-f3cf-4338-b11d-79b0ce963177.png)